cli/bin/grain.js

```diff
@@ -143,6 +143,7 @@ program
     "--verbose",
     "print critical information at various stages of compilation"
   )
+  .graincOption("--format-in-place", "reformat in place")
```

Comment:
I think you can move the option to the `format` command so it only applies when that subcommand is active.